src/components/MultiTextFieldBase/MultiTextFieldBase.tsx

```diff
@@ -85,27 +88,27 @@ const MultiSelectBase = React.forwardRef<HTMLInputElement, Props & InputProps &
     const chips = useMemo(
       () => (
         <>
-          {selectedOptions?.map((option, index) => (
+          {selectedOptions?.map((option: any, index: number) => (
             <span key={generateUniqueID('chip' + index)} css={chipStyle()}>
               <Chip
                 onClear={!(locked || disabled) ? () => onOptionDelete(option) : undefined}
                 fill="lightGrey"
                 dataTestId={`chip_${index}`}
               >
                 <div
-                  title={option.label}
+                  title={typeof option === 'string' ? option : option.label}
                   css={chipContent({
                     maxWidth: TextfieldRef.current?.getBoundingClientRect().width,
                   })}
                 >
-                  {option.label}
+                  {typeof option === 'string' ? option : option.label}
                 </div>
               </Chip>
             </span>
           ))}
         </>
       ),
-      [disabled, locked, onOptionDelete, selectedOptions]
+      [TextfieldRef, disabled, locked, onOptionDelete, selectedOptions]
```

Comment:
Since this wasn't here before but the relevant code didn't change, are we sure we need this?